openapi/openapi.yaml

```diff
@@ -191,6 +191,28 @@ paths:
         'default':
           $ref: '#/components/responses/Errors'
 
+  '/v3/collections/{namespace}/{name}/versions/{version}/move/{source_path}/{destination_path}':
+    parameters: #tag
+      - $ref: '#/components/parameters/CollectionNamespaceName'
+      - $ref: '#/components/parameters/CollectionName'
+      - $ref: '#/components/parameters/SemanticVersion'
+      - $ref: '#/components/parameters/DistributionBasePath'
+      - $ref: '#/components/parameters/DistributionBasePath'
+    post:
+      summary: >
+        Queues tasks to move CollectionVersion
+        from source Repository to destination Repository
+        using the Repository DistributionBasePath.
+      operationId: moveCollectionVersion
+      tags:
+        - Collections
+      parameters: []
+      responses:
+        '200':
```

Comment:
Posting to this endpoint creates async tasks that (possibly) have not finished at the end of the request?

Could be more appropriate to return 202 for that case  (https://geemus.gitbooks.io/http-api-design/content/en/responses/return-appropriate-status-codes.html)

Reply:
yeah a 202 makes sense